src/script/components/resource-hub-cards.ts

```diff
@@ -0,0 +1,61 @@
+import { html } from 'lit-element';
+
+interface CardData {
+  imageUrl: string;
+  title: string;
+  description: string;
+  linkUrl: string;
+}
+
+const resourceCards: Array<CardData> = [{
+  imageUrl: "/assets/icons/icon_120.png",
+  title: "Blog",
+  description: "Card description. Lorem ipsum dolor sit amet, consectetur elit adipiscing",
+  linkUrl: ""
+}, {
+  imageUrl: "/assets/icons/icon_120.png",
+  title: "Demo",
+  description: "Card description. Lorem ipsum dolor sit amet, consectetur elit adipiscing",
+  linkUrl: ""
+}, {
+  imageUrl: "/assets/icons/icon_120.png",
+  title: "Components",
+  description: "Card description. Lorem ipsum dolor sit amet, consectetur elit adipiscing",
+  linkUrl: ""
+}, {
+  imageUrl: "/assets/icons/icon_120.png",
+  title: "Documentation",
+  description: "Card description. Lorem ipsum dolor sit amet, consectetur elit adipiscing",
+  linkUrl: ""
```

Comment:
Assumption is that we will be using links to navigate. Easy to change if not the case